catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR.jsonnet

```diff
@@ -0,0 +1,534 @@
+    in each pixel.  The structural change method partitions the time-series into
+    segments of similar slope values, with statistically significant
+    break-points indicating perturbations to the prior trajectory.  The break
+    point trends analysis suite relies on structural break methods, resulting in
+    the identification of the number and timing of breaks in the time-series,
+    and the significance of each segment. We produce the following statistics:
+    1) for each component, each year, the presence/absence of breaks, 2) the
+    slope, p-value, and standard error of the segment occurring in each year, 3)
+    the overall model R2 (quality of model fit to the temporal profile), and 4)
+    an index, Total Change Intensity. This index reflects the total amount of
+    change occurring across components in that pixel. The linear and structural
+    change methods generally agreed on patterns of change, but the latter found
+    breaks more often, with at least one break point in most pixels. The
+    structural change model provides more robust statistics on the significant
+    minority of pixels with non-monotonic trends, while detrending some
+    interannual signal potentially superfluous from a long-term
+    perspective. Trends products can be downloaded from the [Multi-Resolution
+    Land Characteristics Consortium](https://www.mrlc.gov/data).
+
+    See also:
+
+    * Rigge, M., C. Homer, L. Cleeves, D. K. Meyer, B. Bunde, H. Shi, G. Xian,
+      S. Schell, and M. Bobo. 2020. Quantifying western U.S. rangelands as
+      fractional components with multi-resolution remote sensing and in situ
+      data. Remote Sensing 12.
+      [doi:10.3390/rs12030412](https://doi.org/10.3390/rs12030412)
+
+    * Rigge, M., C. Homer, H. Shi, D. Meyer, B.
+      Bunde, B. Granneman, K. Postma, P. Danielson, A. Case, and G. Xian. 2021.
+      Rangeland Fractional Components Across the Western United States
+      from 1985 to 2018. Remote Sensing 13:813.
+      [doi:10.3390/rs13040813](https://doi.org/10.3390/rs13040813)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'climate_change',
+    'disturbance',
+    'landsat_derived',
+    'nlcd',
+    'rangeland',
+    'trends',
+  ],
+  providers: [
+    ee.producer_provider(
+      'United States Geological Survey and Bureau of Land Management',
+      'https://www.mrlc.gov/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(
+      -125.0683594, 28.45903302, -101.0742188, 49.32512199,
+      '1985-01-01T00:00:00Z',
+      '2021-12-31T00:00:00Z'),
+  'gee:interval': {
+    type: 'cadence',
+    unit: 'year',
+    interval: 1,
+  },
+  summaries: {
+    gsd: [30],
+    // TODO(schwehr): Use units.thing for gee:units 
+    'eo:bands': [
+      {
+        name: 'annual_herbaceous_break_point',
+        description: 'Presence or absence of structural breaks in each component for each year',
+        'gee:units': 'units_counts',
```

Comment:
Don't quote the right hand side of the `:`.   Those are variables.

`'units_counts'` → `units_counts`

e.g.

`'gee:units': units_counts,`